setup.py

```diff
@@ -70,7 +71,9 @@ class TestCommand(Command):
 
     def get_args(self):
         """Return args to be used in test command."""
-        return '--size %s --type %s' % (self.size, self.type)
+        tmpl = Template("--size ${size} --type ${type}")
```

Comment:
@rmotitsuki can you add `packages=[]` same idea as described on this comment https://github.com/kytos-ng/flow_stats/pull/18#discussion_r852213307?


Otherwise it can result in issues when pip installing with py39

```
  ERROR: Can't roll back amlight-sdntrace-cp; was not uninstalled
error: subprocess-exited-with-error

× python setup.py develop did not run successfully.
│ exit code: 1
╰─> [16 lines of output]
    /home/viniarck/repos/sdntrace_cp/.direnv/python-3.9.12/lib/python3.9/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requi
rements should be satisfied by a PEP 517 installer.
      warnings.warn(
    error: Multiple top-level modules discovered in a flat-layout: ['settings', 'utils', 'automate', 'main'].

    To avoid accidental inclusion of unwanted files or directories,
    setuptools will not proceed with this build.

    If you are trying to create a single distribution with multiple modules
    on purpose, you should not rely on automatic discovery.
    Instead, consider the following options:

    1. set up custom discovery (`find` directive with `include` or `exclude`)
    2. use a `src-layout`
    3. explicitly set `py_modules` or `packages` with a list of names

    To find more information, look for "package discovery" on setuptools docs.
    [end of output]

note: This error originates from a subprocess, and is likely not a problem with pip.
```